### PR TITLE
Address Code Review Issues

### LINKED ISSUE #27: Hardcode `get_flags` to 384 to better match the spec
See the full code review thread for details.
              
              question: for your use case, do you actually need support for arbitrary `n` or do you have a fixed curve? Asking because the spec hard-codes a specific value for `n`.

_Originally posted by @marsella in https://github.com/GaloisInc/ckzg-eip-4844-verification/pull/23#discussion_r1781171765_

### LINKED ISSUE #28: Switch from list comprehension to map
Since the `map` is slightly more obvious that we are just applying a conversion function to all elements in the list.

              note: you could also `map toIntegral powers'` but I don't have a preference either way.

_Originally posted by @marsella in https://github.com/GaloisInc/ckzg-eip-4844-verification/pull/23#discussion_r1781512088_

### LINKED ISSUE #31: Add missing check to `decompress_G1`
todo: The spec also checks that the actual bytes of `z` are the point-at-infinity bytes. I recommend you either do the same check here or add a comment to explain why you don't.

_Originally posted by @marsella in https://github.com/GaloisInc/ckzg-eip-4844-verification/pull/23#discussion_r1781324089_

### LINKED ISSUE #32: Fix implementation of `int_to_bit`
suggestion: expand this doc to explain the assumptions from your use case below. E.g. I think the bit shift plus `&&` means that all the top-level bits are 0s, right? So this takes an `n`-bit integer that's either 0 or 1.

Noting because I think I would expect a function of this shape to return 0 if 0 and 1 if it's anything greater than 0 and that's not what'll happen here.

_Originally posted by @marsella in https://github.com/GaloisInc/ckzg-eip-4844-verification/pull/23#discussion_r1781181336_

### LINKED ISSUE #33: Add url doc link for `g1_multi_exp`
note: I think this is not on you but it's weird that the spec doesn't actually provide a function body for this.

_Originally posted by @marsella in https://github.com/GaloisInc/ckzg-eip-4844-verification/pull/23#discussion_r1781162790_
            
            
 Link to `bls.py` is here: https://github.com/ethereum/consensus-specs/blob/85e2452301382a9d099df16f38deac8178355358/tests/core/pyspec/eth2spec/utils/bls.py#L262

### PR DESCRIPTION
**Pull request by b13decker**
Closes #27, closes #28, closes #31, closes #32, closes #33.

### CODE REVIEW

**Review by marsella — APPROVED** [review]
These changes look good, thank you!

File: spec/Spec/BlsHelpers.cry [hunk 1]
```diff
@@ -43,12 +43,19 @@ bls_div x y = toIntegral (div_mod x' y') where
  * NOTE: this Cryptol function requires n >= 2, which is not the same constraint on the Python function,
```
Comment (marsella) [inline spec/Spec/BlsHelpers.cry:43]: todo: fix this doc to reflect your changes

**Review by marsella — APPROVED** [review]
One tiny request, but otherwise this looks good!

File: spec/Spec/BlsHelpers.cry (hunk 1 above)
Comment (b13decker) [reply spec/Spec/BlsHelpers.cry:43]: Thank you ....

Comment (b13decker) [reply spec/Spec/BlsHelpers.cry]: Fixed: 1c71b2cdb298ed6f94181fb450ba78de5dc0c0ca

**Pull request merged**